src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
The check for `num === undefined` is correct for determining when to return the accumulated sum. However, consider using `arguments.length === 0` to check if the function was called without arguments, which is a more explicit way to handle this scenario.